Source/EasyNetQ/IEventBus.cs

```diff
@@ -1,102 +1,106 @@
-﻿﻿using System;
-﻿using System.Collections.Concurrent;
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using EasyNetQ.Logging;
 
-﻿namespace EasyNetQ
+namespace EasyNetQ
 {
     /// <summary>
-    /// An internal pub-sub bus to distribute events within EasyNetQ
+    ///     An internal pub-sub bus to distribute events within EasyNetQ
     /// </summary>
     public interface IEventBus
     {
+        /// <summary>
+        ///     Publishes the event
+        /// </summary>
+        /// <param name="event">The event</param>
+        /// <typeparam name="TEvent">The event type</typeparam>
         void Publish<TEvent>(TEvent @event);
```

Comment:
I propose to add base class for all bus events (maybe with single `Endpoint` property) and add generic constraint here for publish and subscribe.